### PR TITLE
add EIP1559 tx button

### PR DESCRIPTION
**Pull request by brad-decker**
Adds this button but only makes it visible when the network supports EIP-1559

https://user-images.githubusercontent.com/4448075/123334988-75a37280-d509-11eb-90ac-8da063579ec1.mov

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -138,6 +138,14 @@ <h4 class="card-title">
                 >
                   Send
                 </button>
+                <button
+                class="btn btn-primary btn-lg btn-block mb-3"
+                id="sendEIP1559Button"
+                disabled
+                hidden
+              >
```
Comment (ryanml) [inline src/index.html:146]: nit: Can we fix the indentation here?
```
<button
  class="btn btn-primary btn-lg btn-block mb-3"
  id="sendEIP1559Button"
  disabled
  hidden
>
```

Comment (brad-decker) [reply src/index.html]: Sorry about that, yes!

**Review by adonesky1 — DISMISSED** [review]
LGTM!

File: src/index.html [hunk 2]
```diff
@@ -138,6 +138,15 @@ <h4 class="card-title">
                 >
                   Send
                 </button>
+
+                <button
+                  class="btn btn-primary btn-lg btn-block mb-3"
+                  id="sendEIP1559Button"
+                  disabled
```
Comment (NiranjanaBinoy) [inline src/index.html:145]: If we switch to a network that supports EIP-1559 without connecting to an account new button is enabled state.

Screen Shot 2021-06-24 at 5 52 04 PM

Comment (brad-decker) [reply src/index.html:145]: Responded in latest commit :D thanks @NiranjanaBinoy

**Comment by brad-decker** [comment]
@ryanml @NiranjanaBinoy @adonesky1 this has been rebased to get latest lint rules updated :)

**Review by ryanml — APPROVED** [review]
++

**Pull request merged**